docs/source/management/role.rst

```diff
@@ -23,12 +23,16 @@ Operation defines the application logic action, for example::
 - write
 - order
 
+Note that in any of the above stanzas, a '*' is taked to mean 'all'.
```

Comment:
Will this format weird? I see it's italicized. I haven't generated the docs yet locally to check for sure.

Reply:
That's a good thought, my RST preview didn't show anything, but will test locally just to be sure.